libs/src/services/creatorNode/index.js

```diff
@@ -435,6 +445,60 @@ class CreatorNode {
     this.authToken = null
   }
 
+  /**
+   * Gets and returns the clock values across the replica set for the wallet in userStateManager.
+   * @returns {Object[]} Array of objects with the structure:
+   *
+   * {
+   *  type: 'primary' or 'secondary',
+   *  endpoint: <Content Node endpoint>,
+   *  clockValue: clock value (should be an integer) or null
+   * }
+   *
+   * 'clockValue' may be null if the request to fetch the clock value fails
+   */
+  async getClockValuesFromReplicaSet () {
```

Comment:
`getClockValuesFromReplicaSet()` is addressed in this PR https://github.com/AudiusProject/audius-protocol/pull/1126. can ignore in this PR

Reply:
just merged today -- will update this PR

Reply:
👍